pcluster/pcluster_stack.py

```diff
@@ -317,8 +347,8 @@ def repl(mykey, myr, myval):
         cdk.CfnOutput(self, 'UserLoginUrl', value="".join(["https://", self.account,".signin.aws.amazon.com/console"]), condition=user_condition)
         cdk.CfnOutput(self, 'UserName', value=user.ref, condition=user_condition )
 
-        base_os = cdk.CfnParameter(self, "Operating System", default="alinux2", type="String", allowed_values=['ubuntu1804','alinux2']).value_as_string
-        cdk.CfnOutput(self, 'OS', value=base_os)
+        base_os = cdk.CfnParameter(self, "Operating System", default="alinux2", type="String", allowed_values=['ubuntu1804','alinux2'])
```

Comment:
Any issues with centos? ;)

Reply:
My frugality? Per-second billing! 

Ok, ok. I'll put the other OS options in there. Use at your own risk

Reply:
Done.